docs/ciphers.rst

```diff
@@ -156,9 +153,9 @@ recommendations with sources of expert guidance on ciphersuites and other
 cryptographic parameters. We're grateful to everyone who contributed
 suggestions. The recommendations we received are available at
 
-https://github.com/letsencrypt/letsencrypt/wiki/Ciphersuite-guidance
+https://github.com/certbot/certbot/wiki/Ciphersuite-guidance
```

Comment:
Do we want to start changing these URLs yet? If we merge this now, our documentation will be broken.

We can either:

1. Wait to merge this PR until we move the repo.
2. Make these changes in a later PR, perhaps even after the 0.6.0 release. Leaving references to `https://github.com/letsencrypt/letsencrypt/*` shouldn't break anything.

I'm personally in favor of 2.